pkg/catalog/catalog.go

```diff
@@ -87,13 +87,19 @@ func VersionExists(client *occlient.Client, componentType string, componentVersi
 }
 
 // getDefaultBuilderImages returns the default builder images available in the
-// openshift namespace
+// openshift namespace and current namespace
 func getDefaultBuilderImages(client *occlient.Client) ([]CatalogImage, error) {
 	imageStreams, err := client.GetImageStreams(occlient.OpenShiftNameSpace)
 	if err != nil {
-		return nil, errors.Wrap(err, "unable to get Image Streams")
+		return nil, errors.Wrapf(err, "unable to get Image Streams from %s namespace", occlient.OpenShiftNameSpace)
 	}
 
+	imageStreams2, err := client.GetImageStreams(client.GetCurrentProjectName())
+	if err != nil {
+		return nil, errors.Wrapf(err, "unable to get Image Streams from %s namespace", imageStreams2)
```

Comment:
this should be `client.GetCurrentProjectName()` instead of `imageStreams2`

Reply:
Ack